doc/create-cloud-resources.rst

```diff
@@ -0,0 +1,145 @@
+.. _create-cluster:
+
+Create your cloud resources
+===========================
+
+BinderHub is built on top of JupyterHub, which uses Kubernetes to manage
+user instances in the cloud. You'll first need to set up some cloud resources,
+and make sure that Kubernetes is running on them, and configure them to work
+properly with one another.
+
+Setting up Kubernetes on `Google Cloud <https://cloud.google.com/>`_
+--------------------------------------------------------------------
+
+.. note::
+
+   Currently, BinderHub only runs on Google Cloud. Support for other cloud
```

Comment:
This is only sortof true. It works on all cloud providers - we only provide instructions for Google Cloud. There's nothing google cloud specific here.

I think we should say:

"BinderHub is built to be cloud agnostic, and can run on various clouds / bare metal. However, here we only provide instructions for Google Cloud. If you would like to help with adding instructions for other cloud providers, please contact us!"